[T7][T11-C4]

pyokagan opened the pull request:
## PRs merged
- [Remove all eclipse project files](https://github.com/CS2103AUG2016-T11-C4/main/pull/1)
- [Fix code style violations and use checkstyle plugin](https://github.com/CS2103AUG2016-T11-C4/main/pull/2)
- [build.gradle cleanups](https://github.com/CS2103AUG2016-T11-C4/main/pull/3)
- [ui: simplify UiPart](https://github.com/CS2103AUG2016-T11-C4/main/pull/4)
- [The Great Rename (Part 1)](https://github.com/CS2103AUG2016-T11-C4/main/pull/5)
- [The Great Rename (Part 2)](https://github.com/CS2103AUG2016-T11-C4/main/pull/6)
- [Tweak code to be more like a task manager (v0.1 edition)](https://github.com/CS2103AUG2016-T11-C4/main/pull/7)
- [User and Developer's guide](https://github.com/CS2103AUG2016-T11-C4/main/pull/8)

kychua commented on docs/UserGuide.md:
```diff
@@ -2,133 +2,411 @@
+
+   ![alt text](./images/userguide/calender.png "")
+
+   ![alt text](./images/userguide/toggle_calender.png "")
```
make sure you've thought through the UI:
* How are the deadline tasks ordered? since it looks like earlier tasks are displayed later here
* Can make better use of colours to distinguish between components 
* Can I switch panes/scroll using the keyboard? 
* The events look hard to read in this pic
* Is it necessary to hide the command bar? 
* Is there feedback for users when they enter commands? If so, where is it displayed?

kychua commented on docs/UserGuide.md:
```diff
@@ -2,133 +2,411 @@
+
+### Show empty time slots : `slot`
+
+Show all empty time slots in a given time period with a given duration.
```
This would be for the PowerSearch feature? The search keywords function seems very basic though... note that it's better to have a few good features than have more half-baked features so make sure any features you add are well-thought-out

kychua reviewed (commented):
@pyokagan @nightism @potatowagon @snehasp13 Some comments added. Please close the PR after reading comments.

Remember to tag V0.1

kychua reviewed (commented):
- naming for boolean: should be `isPresent`
  ![](https://cloud.githubusercontent.com/assets/7569990/19830059/d5b70cb8-9e22-11e6-9c26-405c827600b9.png)
- Remember to check header comments
  ![](https://cloud.githubusercontent.com/assets/7569990/19830060/df7100d8-9e22-11e6-878d-cef3f3cc676f.png)
- Is there a better way of doing this besides using exceptions?
  ![](https://cloud.githubusercontent.com/assets/7569990/19830062/e42567cc-9e22-11e6-9bc8-412b6c121700.png)
- would probably be useful to explain the parsing in the developer guide
- A0147919N did not code anything?

Overall good job!

Commit:
Command: rename the compilation unit ClearCommand to ClearAllCommand

Commit:
add help box

Commit:
make help box close when esc key pressed

Commit:
comments

Commit:
make help box visible when user is inputting a command

Commit:
press <esc> to close msg

pyokagan committed:
ui: add nice icons

pyokagan committed:
Implement "find" command (find by keywords)

pyokagan committed:
ui: small CommandBox bugfix

pyokagan committed:
MainWindow: evenly divide events and deadline/floating task panes

Commit:
add contents for help box

Commit:
remove unsued imports

pyokagan committed:
Merge branch 'filter-rework'

* The user interface has gained some nice icons!

* Commands "list fin" and "list unfin" will now list only finished and
  unfinished tasks respectively.

* A "find" command has been implemented for filtering tasks that match a
  subset of a set of keywords, case-insensitively.

* The menu bar in the UI has been removed, and replaced with a "top
  bar". The top bar will show the current active filter.

* filter-rework:
  MainWindow: evenly divide events and deadline/floating task panes
  ui: small CommandBox bugfix
  Implement "find" command (find by keywords)
  ui: add nice icons
  ui: create TopBar
  logic: expose read-only model view (getModel())
  build.gradle: import fontawesomefx-fontawesome
  model: extract out ReadOnlyModel
  read-only property
  ui: remove menu bar
  view: simplify DarkTheme.css
  filter: implement TaskUnfinishedPredicate
  logic: extend "list" command to take a filter predicate argument
  model: support only one task filter

pyokagan committed:
Hotfix for 'filter-rework'

Turns out that when hiding and showing the MainWindow (e.g. using the
Ctrl-Space feature), all the fontawesome icons become squares. This is
because JavaFX does some funky stylesheet stuff, and the
"-fx-font-family: Open Sans" ends up overriding the glyph-icon font. We
work around this by explicitly setting the font family of glyph-icon to
FontAwesome.

Commit:
resize window

Commit:
TaskBook: add the support for "removing tasks that satisfy a given predicate"

Commit:
WorkingTaskBook: add support for "removing tasks that satisfy a given predicate"

Commit:
Model: add support for "removing tasks that satisfy a given predicate"

Commit:
Command: add ClearFinishedCommand

Commit:
Command: implement ClearFinishedCommand

Commit:
add method for neat printing of localdate time

Commit:
edit format

Commit:
apply changes to event task

pyokagan committed:
Remove unused methods

pyokagan committed:
Merge branch 'pr/167-new_clear_fin'

The "clear" command has been extended. "clear fin" will now clear tasks
IN THE CURRENT FILTERED LIST that have been finished. (Note: "clear"
will clear all tasks both in and not in the filtered list!)

* pr/167-new_clear_fin:
  Remove unused methods
  Command: implement ClearFinishedCommand
  Command: add ClearFinishedCommand
  Model: add support for "removing tasks that satisfy a given predicate"
  WorkingTaskBook: add support for "removing tasks that satisfy a given predicate"
  TaskBook: add the support for "removing tasks that satisfy a given predicate"
  Command: rename the compilation unit ClearCommand to ClearAllCommand

pyokagan committed:
DeveloperGuide: move class diagrams to the "Implementation" section

pyokagan committed:
DeveloperGuide: Update logic class diagram

pyokagan committed:
DeveloperGuide: move logic component sequence diagram to implementation section

pyokagan committed:
DeveloperGuide: update Model component class diagram

Commit:
Write test for deadline and event card

pyokagan committed:
DeveloperGuide: update logic sequence diagram

pyokagan committed:
DeveloperGuide: add Parser information

Commit:
format help contents

Commit:
shift localDateTimeToPrettyString to Utils

pyokagan committed:
Merge branch 'pr/169-ui_local_date_time_format'

Dates and times are now displayed in a human-readable manner in the UI.

* pr/169-ui_local_date_time_format:
  shift localDateTimeToPrettyString to Utils
  Write test for deadline and event card
  apply changes to event task
  edit format
  add method for neat printing of localdate time

pyokagan committed:
Remove more unused code

pyokagan committed:
Implement Autocomplete

Commit:
FootBar: Add total Task counter

Commit:
TaskCounter: add task counter to evnet task

Commit:
TaskCounter: add task counter to deadline task

Commit:
TaskCounter: add task counter to floating task

Commit:
add font

Commit:
set monospace font

Commit:
add font

Commit:
webview

Commit:
remove duplicate code

Commit:
program enter key to launch userguide when helBox is open. this can be done multiple times

pyokagan committed:
Remove EventBus

Currently, the event bus is only used for showing the "Task book last
modified" status in the UI. Yes, the entire event system is used just
for one status message, which is itself not really that useful and, if
really needed, could be re-implemented with much less code.

But that's not the real problem though -- The real problem is that the
even bus system is bad for encapsulation. The "EventsCenter" is a global
singleton, and all listeners for events of type T will receive all of
its events, no matter which object actually sent it. This means that it
is impossible to cleanly separate an instance of the task tracker
application from another.

An example: Let's say we implement a feature that allows the user to run
multiple instances of the application in the same process. We will thus
have two Models, A & B. We will also have two status bars (Task book
last modified status), sA and sB.  However, if only A gets modified,
BOTH sA and sB will update their status, which is totally wrong since
model B was not modified at all.

Furthermore, relying on events means that we need to pepper the code
with event raising functions in order to satisfy what listeners expect.
However, for complex systems, this is a potential source of bugs. With
the current code of Task Tracker, for example, if you try to run "clear
fin", and finished tasks were actually cleared, you will notice the
"Task book last modified" status still does not change at all. This is
because the developer of the code forgot to add in the
"indicateTaskBookChanged()" method call to the relevent code paths.

To conclude, the event bus enforces poor encapsulation and is an extra
source of bugs, so let's remove it.

Commit:
increase width of helpBox

pyokagan committed:
Add SubcommandParserTest

Commit:
shift end tie to be on the same line as start time

Commit:
fix test

pyokagan committed:
Add FileParserTest

pyokagan committed:
EditEventParserTest: increase test coverage

pyokagan committed:
add TaskPredicateParserTest

pyokagan committed:
CommandLineParserTest: improve test coverage

pyokagan committed:
EditFloatingTaskParserTest: improve test coverage

pyokagan committed:
EditDeadlineParserTest: improve test coverage

pyokagan committed:
CommandBoxTest: improve test coverage

pyokagan committed:
Add EventTaskListPaneTest

pyokagan committed:
Add DeadlineTaskListPaneTest

pyokagan committed:
Add FloatingTaskListPaneTest

pyokagan committed:
Add EditEventCommandTest

pyokagan committed:
FxViewUtil: remove dead code

pyokagan committed:
Remove IncorrectCommand

It is not used any more.

pyokagan committed:
Add ListCommandTest

pyokagan committed:
Add DeleteEventCommandTest

pyokagan committed:
Add AddEventCommandTest

pyokagan committed:
ParseException: remove unused methods

pyokagan committed:
Merge branch 'pr/176-remove-eventbus'

The event bus has been removed.

* pr/176-remove-eventbus:
  Remove EventBus

pyokagan committed:
Merge branch 'pr/175-autocomplete'

The command box has gained tab completion functionality. Just press
<tab>.

* pr/175-autocomplete:
  ParseException: remove unused methods
  Add AddEventCommandTest
  Add DeleteEventCommandTest
  Add ListCommandTest
  Remove IncorrectCommand
  FxViewUtil: remove dead code
  Add EditEventCommandTest
  Add FloatingTaskListPaneTest
  Add DeadlineTaskListPaneTest
  Add EventTaskListPaneTest
  CommandBoxTest: improve test coverage
  EditDeadlineParserTest: improve test coverage
  EditFloatingTaskParserTest: improve test coverage
  CommandLineParserTest: improve test coverage
  add TaskPredicateParserTest
  EditEventParserTest: increase test coverage
  Add FileParserTest
  Add SubcommandParserTest
  Implement Autocomplete

pyokagan committed:
EditDeadlineCommand: retain old task's finished field

pyokagan committed:
EditFloatingTaskCommand: retain old task's finished field

pyokagan committed:
Refine EditEventCommand

pyokagan committed:
Name: accept punctuation as well

pyokagan committed:
Merge branch 'pr/178-fix-edit-resets-fin'

Previously, the "edit" command will reset any task's finished status
back to "unfinished". Now, it will retain the finished status.

* pr/178-fix-edit-resets-fin:
  Refine EditEventCommand
  EditFloatingTaskCommand: retain old task's finished field
  EditDeadlineCommand: retain old task's finished field

pyokagan committed:
Merge branch 'pr/179-relax-name-restrictions'

Task names can now store punctuation as well, which should provide much
more flexibility in task names.

* pr/179-relax-name-restrictions:
  Name: accept punctuation as well

pyokagan committed:
Merge branch 'pr/177-make_event_task_space_saving'

* pr/177-make_event_task_space_saving:
  shift end tie to be on the same line as start time

pyokagan committed:
MainWindow: fix empty task selection bug

If there is an empty task selection, we must explicitly clear the task
selections of all task panes rather than just punting.

pyokagan committed:
ui: select tasks with a yellow-ish background

pyokagan committed:
model: Merge IndexPrefix with TaskType

Commit:
helpwindow test

pyokagan committed:
DeveloperGuide: add information on model indexing/filtering/sorting

pyokagan committed:
Add task model diagram

pyokagan committed:
Add TaskBook class diagram

Commit:
add ROBOTOMONO font

pyokagan committed:
Add WorkingTaskBook diagram

pyokagan committed:
Simplify model component diagram

pyokagan committed:
Add Config diagram

pyokagan committed:
Remove EventBus

pyokagan committed:
Fix seq-deleteevent

pyokagan committed:
Update logic parser class diagram

Commit:
finish GUI testing

Commit:
FootBar: TotalTaskCounter revised

* now total task counter counts the total number of the task stored in TaskTracker rather than the tasks listed in UI

Commit:
TaskCounter: add finished floating tasks counter

pyokagan committed:
Update logic parser section

Commit:
TaskCounter: add finished deadline counter

Commit:
TaskCounter: add finished event counter

Commit:
TaskCounter: add overdue deadline counter

Commit:
Ui: polish ui

Commit:
TaskCounter: add today remaining counter

Commit:
fix format helpbox content

Commit:
add release and download image to useguide

Commit:
update summary table

pyokagan committed:
Fix up model diagrams

pyokagan committed:
Add command parser listing

pyokagan committed:
edit

pyokagan committed:
Update Logic comp

pyokagan committed:
Add Command interface diagram

pyokagan committed:
Command interface

pyokagan committed:
Refine Model section

pyokagan committed:
Update Logic

Commit:
add stories for add command, format

pyokagan committed:
Merge branch 'update-devguide'

The developer guide has been updated.

* update-devguide:
  Update Logic
  Refine Model section
  Command interface
  Add Command interface diagram
  Update Logic comp
  edit
  Add command parser listing
  Fix up model diagrams
  Update logic parser section
  Update logic parser class diagram
  Fix seq-deleteevent
  Remove EventBus
  Add Config diagram
  Simplify model component diagram
  Add WorkingTaskBook diagram
  Add TaskBook class diagram
  Add task model diagram
  DeveloperGuide: add information on model indexing/filtering/sorting
  DeveloperGuide: add Parser information
  DeveloperGuide: update logic sequence diagram
  DeveloperGuide: update Model component class diagram
  DeveloperGuide: move logic component sequence diagram to implementation section
  DeveloperGuide: Update logic class diagram
  DeveloperGuide: move class diagrams to the "Implementation" section

Commit:
Filter: update TaskWillHappenTodayPredicate

Commit:
EventTaskCounter: reset the filters used in TaskCounter when tasklist is updated

Now when the event tasks listed in the ui change, the filters used when generating task counter will be updated with the last modified time

Commit:
no message

Commit:
DeadlineTaskCounter: reset the filters used in TaskCounter when tasklist is updated

Now when the deadline tasks listed in the ui change, the filters used when generating task counter will be updated with the last modified time

Commit:
small changes in FloatingTaskListPane

This commit is to make FloatingTaskListPane similar to Deadline/EventTaskListPane

Commit:
del task

Commit:
Add using the interface

Commit:
finish edit portion, finish list

pyokagan committed:
Fix small bug

Commit:
update user guide

pyokagan committed:
Fix fonts

pyokagan committed:
Code cleanup

pyokagan committed:
Merge branch 'pr/170-improved_help_without_table_library'

The help box shown when the "help" command is run has been updated to
show relevant Task Tracker help.

* pr/170-improved_help_without_table_library:
  Code cleanup
  Fix fonts
  Fix small bug
  finish GUI testing
  add ROBOTOMONO font
  helpwindow test
  increase width of helpBox
  program enter key to launch userguide when helBox is open. this can be done multiple times
  remove duplicate code
  webview
  add font
  set monospace font
  add font
  format help contents
  fix test
  resize window
  remove unsued imports
  add contents for help box
  press <esc> to close msg
  make help box visible when user is inputting a command
  comments
  make help box close when esc key pressed
  add help box

pyokagan committed:
Fix command box too big

pyokagan committed:
Merge branch 'style-fixes'

* style-fixes:
  MainWindow: fix empty task selection bug
  model: Merge IndexPrefix with TaskType
  ui: select tasks with a yellow-ish background
  Fix command box too big

pyokagan committed:
UI fixes

pyokagan committed:
Fix tests and warnings

pyokagan committed:
Fix overlong lines

pyokagan committed:
List cell selected -- make text black

Commit:
help box reposition

Commit:
Cleanup code

Commit:
Merge branch 'pr/181-add_task_counter_new'

Task counters have been added.

* pr/181-add_task_counter_new:
  Cleanup code
  Fix overlong lines
  Fix tests and warnings
  UI fixes
  small changes in FloatingTaskListPane
  DeadlineTaskCounter: reset the filters used in TaskCounter when tasklist is updated
  no message
  EventTaskCounter: reset the filters used in TaskCounter when tasklist is updated
  Filter: update TaskWillHappenTodayPredicate
  TaskCounter: add today remaining counter
  Ui: polish ui
  TaskCounter: add overdue deadline counter
  TaskCounter: add finished event counter
  TaskCounter: add finished deadline counter
  TaskCounter: add finished floating tasks counter
  FootBar: TotalTaskCounter revised
  TaskCounter: add task counter to floating task
  TaskCounter: add task counter to deadline task
  TaskCounter: add task counter to evnet task
  FootBar: Add total Task counter

pyokagan committed:
README.md: Add build status/coveralls badge

pyokagan committed:
ui: word wrap task names

pyokagan committed:
ui: Make cell_big_label bigger

pyokagan committed:
Update About Us

pyokagan committed:
Merge branch 'update-aboutus'

The About Us page has been updated.

* update-aboutus:
  Update About Us

pyokagan committed:
Update AboutUs

pyokagan committed:
Merge branch 'more-style-changes'

* more-style-changes:
  ui: Make cell_big_label bigger
  ui: word wrap task names
  List cell selected -- make text black

Commit:
update special features

pyokagan committed:
Connect the status bar to the Config property

pyokagan committed:
Remove setdatadir autocomplete

pyokagan committed:
LogicManager: handle task book move IOException more gracefully

pyokagan committed:
SetDataDirectoryCommand: allow setting relative paths

pyokagan committed:
Merge branch 'setdatadir-fixes'

* setdatadir-fixes:
  SetDataDirectoryCommand: allow setting relative paths
  LogicManager: handle task book move IOException more gracefully
  Remove setdatadir autocomplete
  Connect the status bar to the Config property

pyokagan committed:
pandoc: implement syntax highlighting

pyokagan committed:
pandoc: implement page headers

pyokagan committed:
Add AboutUs

Commit:
add ui images

pyokagan committed:
Fix AboutUs sizes

pyokagan committed:
Fix AboutUs page

Commit:
add in ui images

pyokagan committed:
DevGuide: fix logic diagram

Commit:
update appendix

Commit:
fix broken image link

Commit:
fix broken image link

Commit:
fix image link

pyokagan committed:
Add Cover Page

pyokagan committed:
Update diagrams

pyokagan committed:
Merge tag 'pr/183-final_userguide-v1'

* tag 'pr/183-final_userguide-v1':
  fix image link
  fix broken image link
  fix broken image link
  add in ui images
  add ui images
  update special features
  help box reposition
  update user guide
  finish edit portion, finish list
  Add using the interface
  del task
  add stories for add command, format
  update summary table
  add release and download image to useguide
  fix format helpbox content

pyokagan committed:
Merge branch 'update-devguide2'

* update-devguide2:
  Update diagrams
  Add Cover Page
  DevGuide: fix logic diagram
  Fix AboutUs page
  Fix AboutUs sizes
  Add AboutUs
  pandoc: implement page headers
  pandoc: implement syntax highlighting

pyokagan committed:
Test edit

pyokagan committed:
Merge tag 'pr/191-update_ugdg-v1' into docs

* tag 'pr/191-update_ugdg-v1':
  update appendix

pyokagan committed:
Update DeveloperGuide.md

Commit:
Test: addFloatingTaskCommand test code

pyokagan committed:
Update DeveloperGuide.md

Commit:
Test: addDeadlineCommand test code

pyokagan committed:
Update DeveloperGuide.md

Commit:
Update UserGuide.md

add auto complete, finish userguide

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

Commit:
Update UserGuide.md

fix summary table, add set data dir

Commit:
Update UserGuide.md

add points to note and how to

pyokagan committed:
Update DeveloperGuide.md

Commit:
Update UserGuide.md

Commit:
Update UserGuide.md

pyokagan committed:
Update UserGuide.md

pyokagan committed:
Update UserGuide.md

Commit:
Update UserGuide.md

fix TOC links

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

Commit:
Update UserGuide.md

remove <kbd>

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Update DeveloperGuide.md

pyokagan committed:
Create Acknowledgements

pyokagan committed:
Rename Acknowledgements to Acknowledgements.md

pyokagan committed:
Update Acknowledgements.md

pyokagan committed:
Add acknowledgments

pyokagan committed:
Update DeveloperGuide.md

Commit:
Update UserGuide.md

inclrease level of how to

Commit:
Test: editDeadlineCommand test code

Commit:
Test: editFloatingTaskCommand test code

Commit:
Test: markDeadlineTaskFinishedCommand test code

Commit:
Test: markFloatingTaskFinishedCommand test code

pyokagan committed:
Update UserGuide.md

pyokagan committed:
Update UserGuide.md

pyokagan committed:
Update UserGuide.md

Commit:
implement undo/redo

Commit:
Update DeveloperGuide.md

formatting

pyokagan committed:
Update UserGuide.md

pyokagan committed:
Update UserGuide.md

Commit:
add help ui pic with monospaced font

Commit:
Delete 12.png

Commit:
Add files via upload

pyokagan committed:
Update DeveloperGuide.md

Commit:
Display feedback to user for undo redo comand

pyokagan committed:
Fix tests

pyokagan committed:
Merge branch 'pr/193-display_command_reult_for_undo'

* pr/193-display_command_reult_for_undo:
  Fix tests
  Display feedback to user for undo redo comand

Commit:
Test: ClearFinishedCommand test code

Commit:
Update HelpWindow.java

pyokagan committed:
Merge branch 'pr/195-test_code'

* pr/195-test_code:
  Test: ClearFinishedCommand test code
  Test: markFloatingTaskFinishedCommand test code
  Test: markDeadlineTaskFinishedCommand test code
  Test: editFloatingTaskCommand test code
  Test: editDeadlineCommand test code
  Test: addDeadlineCommand test code
  Test: addFloatingTaskCommand test code

pyokagan committed:
Add files via upload

pyokagan committed:
Update README.md

pyokagan committed:
Use Task Tracker screenshot on cover page.

pyokagan committed:
Merge branch 'docs'

* docs:
  Use Task Tracker screenshot on cover page.
  Update README.md
  Add files via upload
  Update HelpWindow.java
  Update DeveloperGuide.md
  Add files via upload
  Delete 12.png
  add help ui pic with monospaced font
  Update UserGuide.md
  Update UserGuide.md
  Update DeveloperGuide.md
  implement undo/redo
  Update UserGuide.md
  Update UserGuide.md
  Update UserGuide.md
  Update UserGuide.md
  Update DeveloperGuide.md
  Add acknowledgments
  Update Acknowledgements.md
  Rename Acknowledgements to Acknowledgements.md
  Create Acknowledgements
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update UserGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update UserGuide.md
  Update UserGuide.md
  Update UserGuide.md
  Update UserGuide.md
  Update UserGuide.md
  Update DeveloperGuide.md
  Update UserGuide.md
  Update UserGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update UserGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Update DeveloperGuide.md
  Test edit
  update appendix

pyokagan committed:
Update collate files for v0.5

pyokagan committed:
Adjust headers of collated files for v0.5

ss-prab committed:
Update A0147919N.md

Commit:
Update A0143637A.md

add dev guide code

ss-prab committed:
Update A0147919N.md

pyokagan committed:
TaskTracker v0.5

pyokagan committed:
Merge tag 'pr/198-potatowagon-patch-1-v1'

* tag 'pr/198-potatowagon-patch-1-v1':
  Update A0143637A.md

pyokagan committed:
Tweak collate

pyokagan committed:
Merge tag 'pr/197-main-collate-v2'

* tag 'pr/197-main-collate-v2':
  Update A0147919N.md

pyokagan committed:
Tweak unused

pyokagan committed:
Merge tag 'pr/199-test-collate-v2'

* tag 'pr/199-test-collate-v2':
  Update A0147919N.md

pyokagan committed:
Tweak collate

pyokagan committed:
Tweak collate

pyokagan committed:
Tweak collate codes again

Commit:
manual testing script

Commit:
unused userguide

Commit:
no message

pyokagan committed:
Merge tag 'pr/202-mingyang_unused-v1'

* tag 'pr/202-mingyang_unused-v1':
  unused userguide

Commit:
no message

Commit:
no message

pyokagan committed:
Merge tag 'pr/201-test_manual-v4'

* tag 'pr/201-test_manual-v4':
  no message
  no message
  no message
  manual testing script

pyokagan committed:
TaskTracker v0.5
